src/main/java/com/kaleido/service/amazonaws/s3/S3Configuration.java

```diff
@@ -0,0 +1,36 @@
+package com.kaleido.service.amazonaws.s3;
+
+import javax.inject.Named;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import com.kaleido.config.Constants;
+
+import liquibase.util.StringUtils;
+
+@Configuration
+public class S3Configuration {
+    private final Logger log = LoggerFactory.getLogger(S3Configuration.class);
+    @Named(Constants.PLATEMAP_BUCKET_SYSTEM_KEY)
```

Comment:
```suggestion
    @Value("${Constants.PLATEMAP_BUCKET_SYSTEM_KEY:kaleido_platemaps}")
```
I was incorrect earlier in suggesting using `@named`, `@value` is the correct way to get an environment variable

You will then need to add variables to: `src/main/resources/config/application-prod.yml`

Reply:
@wfowlks Thanks. I have changed to Value annotation and added in application-prod.yml